pkg/catalog/catalog.go

```diff
@@ -137,11 +221,11 @@ func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
 }
 
 // SliceSupportedTags splits the tags in to fully supported and unsupported tags
-func SliceSupportedTags(catalogImage CatalogImage) ([]string, []string) {
+func SliceSupportedTags(catalogImage ComponentType) ([]string, []string) {
```

Comment:
shouldn't the argument name also be `catalogType` then?

Reply:
Yes, my mistake :)